spec/Serialization.cry

```diff
@@ -17,9 +17,11 @@ import BLSFieldExtras (sqrt_fp, is_square_fp, sqrt_fp2, is_square_fp2)
 
 //// Serialization (with compression)
 
+// TODO: draft 7 changed this
```

Comment:
Do we have a plan for this TODO?  Is it that the code version we analyse is compliant with an earlier version of the spec that the v9 we used?

Reply:
The code is compliant with the spec, but the spec is weird: there are two versions of sign, used at different places. The code does both at once (see sgn0.saw). At the moment I forgot which is used where.

Reply:
I checked and the spec says serialization should be done using the ZCash format, so what's in Serialization.cry is correct.